test/store/notifications/notifications.reducer.test.js

```diff
@@ -0,0 +1,29 @@
+import { TestUtils } from '../../test-utils';
+import { LevelTypes, notificationReducer } from '../../../src/store/notifications/notifications.reducer';
+import { setNotification } from '../../../src/store/notifications/notifications.action';
+
+
+describe('notificationReducer', () => {
+	const setup = (state) => {
+		return TestUtils.setupStoreAndDi(state, {
+			notifications: notificationReducer
+		});
+	};
+
+	it('initiales the store with default values', () => {
+		const store = setup();
+		expect(store.getState().notifications.notification).toBeNull();
+	});
+
+	it('sets the \'notification\' property', () => {
+		const store = setup();
+		const notification = {
+			message: 'foo',
+			level: LevelTypes.INFO,
+			permanent: false
+		};
+
+		setNotification(notification);
```

Comment:
Would suggest something like 'emitN...', 'publishN...',or 'fireN...' (because it's more like an event)'